tests/integration/awslambda/test_lambda.py

```diff
@@ -171,12 +171,12 @@ def fixture_snapshot(snapshot):
             "$..Environment",  # missing
             "$..HTTPStatusCode",  # 201 vs 200
             "$..Layers",
-            "$..SnapStart",  # FIXME
+            "$..SnapStart",
         ],
     )
 else:
     pytestmark = pytest.mark.skip_snapshot_verify(
-        paths=["$..CodeSize", "$..SnapStart"],  # FIXME
+        paths=["$..CodeSize"],  # FIXME
     )
```

Comment:
we can actually just remove codesize as well and add it to the "global" transformers above

Reply:
Good catch 👍 
I just removed it because `CodeSize` is already part of the `lambda_api()` transformer.